proto/v1/rollout_service.proto

```diff
@@ -266,6 +266,14 @@ message Plan {
     optional RollbackDetail rollback_detail = 6;
 
     map<string, string> ghost_flags = 7;
+
+    message PreUpdateBackupDetail {
+      // The database for keeping the backup data.
+      // Format: instances/{instance}/databases/{database}
+      string database = 1;
+    }
+    // If set, we will create a task backing up data before the DML task is executed.
+    optional PreUpdateBackupDetail pre_update_backup_detail = 8;
```

Comment:
This should be manually selected by the user via UI?

Reply:
Yes, and this is an optional setting on DML. It should not be highlighted.